### ISSUE TITLE
Gems should be vendored for spec tests

### ISSUE THREAD

**Issue by ismith** [colinsurprenant/redstorm#102]
One way to do this would be to run `redstorm bundle development`, but probably better is to add .bundler/config with

```
BUNDLE_PATH: vendor
BUNDLE_DISABLE_SHARED_GEMS: "1"
```

and then you can just `bundle install ; bundle exec rake spec` as usual.

**Comment by colinsurprenant** [comment]
yes, that's what I personally do, add this `~/.bundler/config`. we should document it.
